department-of-veterans-affairs/caseflow
# Replaced heading styling w/ regular p tag and bold styling

Resolves [CASEFLOW-993](https://vajira.max.gov/browse/CASEFLOW-993)

### Description
Replaced the h4 tag for a p tag with exact styling used on h4 tags.

### Acceptance Criteria
- [x] JAWS correctly reads Case Details Headings as intended.

### Testing Plan
1. Go to Case Details page.
2. Run JAWS.
3. Activate 'Headings' Mode.
4. Verify that JAWS reads the heading in the correct order.
